test-infra/prow/config.yaml

```diff
@@ -205,6 +205,27 @@ periodics:
        - name: compute-image-tools-test-service-account
          secret:
            secretName: compute-image-tools-test-service-account
+ - name: ci-windows-upgrade-e2e-tests
+   interval: 6h
+   agent: kubernetes
+   spec:
+     containers:
+     - image: gcr.io/compute-image-tools-test/gce-windwos-upgrade-tests:latest
```

Comment:
typo? (windwos)

Reply:
oooouch, it's too late... let me create another PR...